app/main/actions/profile-actions.js

```diff
@@ -48,3 +50,21 @@ export function completionsFor(text: string,
     },
   };
 }
+
+export function createBrowserWindow(browserWindow: electron.BrowserWindow): ProfileAction {
+  return {
+    type: profileActionTypes.DID_CREATE_BROWSER_WINDOW,
+    payload: {
+      browserWindow,
+    },
+  };
+}
+
+export function closeBrowserWindow(browserWindow: electron.BrowserWindow): ProfileAction {
```

Comment:
I think overloading action / command names is going to get more confusing as time goes on (DID_CLOSE_BROWSER_WINDOW and closeBrowserWindow), and I find the pattern used for IPC communication with 'new-window' / createBrowserWindow easier to follow.   That is, the ipc communication is using plain event listeners on server side, and the listener is in charge of emitting a profileAction (takes profileCommand out of the equation).  Why is closing a window a profile-command but opening one isn't?